core/tools/zenoh/bootstrap.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# Immediately exit on errors
+set -e
+
+PROJECT_NAME="zenoh"
+VERSION="1.0.0"
+BINARIES=(
+  "zenoh"
+  "zenoh-plugin-webserver"
+  "zenoh-backend-filesystem"
+)
+
+echo "Installing project $PROJECT_NAME and friends with version $VERSION"
+
+# Step 1: Prepare the download URL
+
+ARCH="$(uname -m)"
+case "$ARCH" in
+  x86_64 | amd64)
+    TOOLCHAIN="x86_64-unknown-linux-musl"
+    ;;
+  armv7l | armhf)
+    TOOLCHAIN="armv7-unknown-linux-gnueabihf"
+    ;;
+  aarch64 | arm64)
+    TOOLCHAIN="aarch64-unknown-linux-musl"
+    ;;
+  *)
+    echo "Architecture: $ARCH is unsupported, please create a new issue on https://github.com/bluerobotics/BlueOS/issues"
+    exit 1
+    ;;
+esac
+echo "For architecture $ARCH, using toolchain $TOOLCHAIN"
+
+# Step 2: Prepare the installation and tools paths
+
+if [ -n "$VIRTUAL_ENV" ]; then
+    BIN_DIR="$VIRTUAL_ENV/bin"
+else
+    BIN_DIR="/usr/bin"
+fi
+
+BINARY_PATH="$BIN_DIR/$PROJECT_NAME"
+
+mkdir -p "$BINARY_PATH"
+
+# Step 3: Download and install the binaries
+echo "Downloading binaries..."
+
+DOWNLOAD_FOLDER="/tmp/zenoh_and_friends"
+mkdir -p "$DOWNLOAD_FOLDER"
+for BINARY in "${BINARIES[@]}"; do
+  URL="https://download.eclipse.org/zenoh/${BINARY}/${VERSION}/${BINARY}-${VERSION}-${TOOLCHAIN}-standalone.zip"
+  echo " - Download: ${URL}"
+  wget -q "$URL" -O "${DOWNLOAD_FOLDER}/${BINARY}.zip"
+done
+echo "Downloaded all binaries, now installing to $BINARY_PATH"
+
+cd "$DOWNLOAD_FOLDER"
+unzip "*.zip"
+rm -- ./*.zip
+mv -- ./* "$BINARY_PATH"
+cd - && rm -rf "$DOWNLOAD_FOLDER"
+
+echo "Install complete!"
```

Comment:
```suggestion

for BINARY in "${BINARIES[@]}"; do
  echo "Installed binary type: $(file "$(which "$BINARY_PATH/$BINARY")")"
done

echo "Finished installing $PROJECT_NAME"
```

Reply:
Here when building with the suggested code I'm getting this

```
#16 24.88 Installed binary type: cannot open `' (No such file or directory)
#16 24.88 Installed binary type: cannot open `' (No such file or directory)
#16 24.88 Installed binary type: cannot open `' (No such file or directory)
#16 24.88 Finished installing zenoh
```

Reply:
I see, just make it static, then:
```bash
cd "$DOWNLOAD_FOLDER"
unzip "*.zip"
rm -- ./*.zip
INSTALLED_FILES=(*)
mv -- ./* "$BINARY_PATH"
cd - && rm -rf "$DOWNLOAD_FOLDER"

for BINARY in "${INSTALLED_FILES[@]}"; do
  echo "Installed binary type: $(file "$(which "$BINARY_PATH/$BINARY")")"
done

echo "Finished installing $PROJECT_NAME"
```